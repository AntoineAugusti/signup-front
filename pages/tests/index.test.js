import ReactTestRenderer from 'react-test-renderer'
import Index from '../../pages/about'

require('../../lib/tests/local-storage') // eslint-disable-line import/no-unassigned-import

describe('pages | index', () => {
  describe('render', () => {
    const renderer = ReactTestRenderer.create(<Index />)
    it('should be defined', () => {
      expect(renderer).toBeDefined()
    })
    it('should match snapshot', () => {
      expect(renderer).toMatchSnapshot()
    })
  })
})
