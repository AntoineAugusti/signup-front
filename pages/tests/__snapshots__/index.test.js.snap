// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages | index render should match snapshot 1`] = `
<div
  className="page"
>
  <header>
    <nav
      className="nav"
    >
      <div
        className="nav__container"
      >
        <a
          className="nav__link"
          href="https://api.gouv.fr/"
        >
          <img
            alt="Accueil de signup.api.gouv.fr"
            className="nav__logo"
            src="https://api.gouv.fr/img/logo_api.svg"
          />
        </a>
        <ul
          className="nav__links"
        >
          <li>
            <a
              href="http://localhost:3022/oauth/authorize?client_id=82546188522214c3577d35c283ce8842786649b35a026a9d44908037a597f29b&redirect_uri=http%3A%2F%2Flocalhost%3A3000&scope=&response_type=token&state="
            >
              Se connecter
            </a>
          </li>
        </ul>
      </div>
    </nav>
  </header>
  <main>
    <section
      className="section-grey"
      id={null}
    >
      <div
        className="container text-center"
      >
        <h2>
          Vous devez vous connecter avant de continuer
        </h2>
        <a
          className="button large"
          href="http://localhost:3022/oauth/authorize?client_id=82546188522214c3577d35c283ce8842786649b35a026a9d44908037a597f29b&redirect_uri=http%3A%2F%2Flocalhost%3A3000&scope=&response_type=token&state="
        >
          Se connecter
        </a>
      </div>
    </section>
  </main>
  <footer
    className="footer"
  >
    <div
      className="container"
    >
      <div
        className="jsx-696190668 footer__logo"
      >
        <a
          className="jsx-696190668"
          href="https://api.gouv.fr/"
        >
          <img
            alt="Logo api.gouv.fr"
            className="jsx-696190668"
            src="/static/images/logo-api.gouv.fr-white.png"
          />
        </a>
        <ul
          className="jsx-696190668 footer__social"
        >
          <li
            className="jsx-696190668"
          >
            <a
              className="jsx-696190668"
              href="https://twitter.com/BetaGouv"
              title="Twitter"
            >
              <img
                alt="Twitter"
                className="jsx-696190668 icon icon-twitter"
                src="/static/images/social/twitter.svg"
              />
            </a>
          </li>
          <li
            className="jsx-696190668"
          >
            <a
              className="jsx-696190668"
              href="https://github.com/betagouv"
              title="Github"
            >
              <img
                alt="Github"
                className="jsx-696190668 icon icon-github"
                src="/static/images/social/github.svg"
              />
            </a>
          </li>
          <li
            className="jsx-696190668"
          >
            <a
              className="jsx-696190668"
              href="mailto:contact@particulier.api.gouv.fr"
              title="Nous Ã©crire un mail"
            >
              <img
                alt="Email"
                className="jsx-696190668 icon icon-mail"
                src="/static/images/social/email.svg"
              />
            </a>
          </li>
        </ul>
      </div>
      <ul
        className="footer__links"
      >
        <li>
          <h2>
            particulier.api.gouv.fr
          </h2>
        </li>
        <li>
          <a
            href="https://status.particulier.api.gouv.fr"
          >
            Status des API
          </a>
        </li>
      </ul>
    </div>
  </footer>
</div>
`;
